pkg/infra/usagestats/service/usage_stats_test.go

```diff
@@ -273,7 +273,7 @@ func TestMetrics(t *testing.T) {
 			})
 			usageStatsURL = ts.URL
 
-			err := uss.sendUsageStats(context.Background())
+			err = uss.sendUsageStats(context.Background())
```

Comment:
Hm.. I would rather keep the shadowing here, since `err` is local to this function.